<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MusicServerPluginWidget</class>
 <widget class="QDialog" name="MusicServerPluginWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>288</width>
    <height>243</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>第三方插件管理</string>
  </property>
  <widget class="QLabel" name="background">
   <property name="geometry">
    <rect>
     <x>4</x>
     <y>4</y>
     <width>280</width>
     <height>235</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="topTitleName">
   <property name="geometry">
    <rect>
     <x>14</x>
     <y>4</y>
     <width>220</width>
     <height>25</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color:&quot;#FFFFFF&quot;;font-weight:bold</string>
   </property>
   <property name="text">
    <string>第三方插件管理</string>
   </property>
  </widget>
  <widget class="QToolButton" name="topTitleCloseButton">
   <property name="geometry">
    <rect>
     <x>262</x>
     <y>6</y>
     <width>20</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>...</string>
   </property>
  </widget>
  <widget class="QWidget" name="backgroundMask" native="true">
   <property name="geometry">
    <rect>
     <x>4</x>
     <y>29</y>
     <width>280</width>
     <height>210</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background:white;</string>
   </property>
   <widget class="MusicServerPluginTableWidget" name="itemTableWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>15</y>
      <width>260</width>
      <height>150</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="confirmButton">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>195</x>
      <y>175</y>
      <width>75</width>
      <height>26</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="focusPolicy">
     <enum>Qt::NoFocus</enum>
    </property>
    <property name="text">
     <string>确定</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="selectAllCheckButton">
    <property name="geometry">
     <rect>
      <x>17</x>
      <y>180</y>
      <width>60</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>全选</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="priorityCheckButton">
    <property name="geometry">
     <rect>
      <x>95</x>
      <y>180</y>
      <width>90</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>插件优先</string>
    </property>
   </widget>
  </widget>
  <zorder>background</zorder>
  <zorder>backgroundMask</zorder>
  <zorder>topTitleName</zorder>
  <zorder>topTitleCloseButton</zorder>
 </widget>
 <customwidgets>
  <customwidget>
   <class>MusicServerPluginTableWidget</class>
   <extends>QTableWidget</extends>
   <header>musicpluginwidget.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
